### PR TITLE
Setup kk

### PR DESCRIPTION
**Pull request by gitkseniya**
## All Submissions:

* [x] Main has been pulled into branch and up to date.

### New Feature Submissions:

* [x] Have you added an explanation of what your changes do?
* [x] Is the formatting correct for each file?

### Testing:

* [x] Have you written new tests?
* [x] Have you successfully ran tests with your changes locally?
* [x] All new and existing tests passed.

**Test coverage** =  100%
-------------------------------------------------------------------------------
Created Classes with attributes reflecting headers on CSV files:

* `Game`
* `Team`
* `GameTeam`

Created corresponding tests for Classes;
* `game_test`
* `team_test`
* `game_team_test`

Created `StatTracker` Class that initializes `Game`, `Team`, and `GameTeam` classes with cvs data

### CODE REVIEW

**Review by r-stemmle — COMMENTED** [review]
Ryan: I have looked at everything. I don't see any issues here. I'd like to merge!

File: lib/game.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    @season       = info[:season].to_i
+    @type         = info[:type]
+    @date_time    = info[:date_time]
+    @away_team_id = info[:away_team_id].to_i
+    @home_team_id = info[:home_team_id].to_i
+    @away_goals   = info[:away_goals].to_i
+    @home_goals   = info[:home_goals].to_i
+    @venue        = info[:venue]
+    @venue_link   = info[:venue_link]
+  end
```
Comment (jaharaclark) [inline lib/game.rb:24]: As a group, are we going to right align our attributes or align them around the "="?

File: lib/game_team.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    @goals                = info[:goals].to_i
+    @shots                = info[:shots].to_i
+    @tackles              = info[:tackles].to_i
+    @pim                  = info[:pim].to_i
+    @powerplayopportunities = info[:powerplayopportunities].to_i
+    @powerplaygoals       = info[:powerplaygoals].to_i
+    @faceoffwinpercentage = info[:faceoffwinpercentage].to_f
+    @giveaways            = info[:giveaways].to_i
+    @takeaways            = info[:takeaways].to_i
+  end
```
Comment (jaharaclark) [inline lib/game_team.rb:34]: As a group, are we going to right align our attributes or align them around the "="?

File: lib/stat_tracker.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+require './lib/game'
+require './lib/team'
+require './lib/game_team'
+require_relative 'csv_loadable'
+
+class StatTracker
+  include CsvLoadable
+
+  attr_reader :games, :teams, :game_teams
```
Comment (jaharaclark) [inline lib/stat_tracker.rb]: Are we going to align our attr_reader horizontally or vertically?

File: lib/team.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+class Team
+  attr_reader :team_id,
+              :franchiseid,
+              :teamname,
+              :abbreviation,
+              :stadium,
+              :link
+
```
Comment (jaharaclark) [inline lib/team.rb:8]: Are we going to align our attr_reader horizontally or vertically?

File: lib/stat_tracker.rb (hunk 3 above)
Comment (gitkseniya) [reply lib/stat_tracker.rb]: I prefer vertically, but i would also prefer we agree on the code first and then make it look "pretty" ;)

Comment (r-stemmle) [reply lib/stat_tracker.rb]: I like vertical

Comment (gunnarrunner) [reply lib/stat_tracker.rb]: I don't mind using either but if people want to use vertical that is cool in my book!

**Review by gunnarrunner — APPROVED** [review]
This looks good. I can follow everything and it makes sense to me I believe this should be pushed up.

**Pull request merged**